qatrack/qa/signals.py

```
@@ -287,3 +291,34 @@ def check_approved_statuses(*args, **kwargs):
+    """When a test list is completed invalidate the unreviewed counts"""
+    cache.delete(settings.CACHE_UNREVIEWED_COUNT)
+    cache.delete(settings.CACHE_UNREVIEWED_COUNT_USER)
+    cache.delete(settings.CACHE_RTS_QA_COUNT)
+    cache.delete(settings.CACHE_RTS_INCOMPLETE_QA_COUNT)
+    cache.delete(settings.CACHE_IN_PROGRESS_COUNT_USER)
+    cache.delete(settings.CACHE_SL_NOTIFICATION_TOTAL)
+
+
+@receiver(post_save, sender=models.UnitTestCollection, dispatch_uid="qa_update_active_unit_test_collections_for_unit_utc_post_save")
+@receiver(post_delete, sender=models.UnitTestCollection, dispatch_uid="qa_update_active_unit_test_collections_for_unit_utc_post_delete")
+def update_active_unit_test_collections_for_unit_utc(sender, instance, **kwargs):
```

Ruff (0.15.18)

[warning] Unused function argument: `sender`

(ARG001)